### PR TITLE
Adding Default quota function to p9admin

### PR DESCRIPTION
**Pull request by mikkergimenez**
This Adds the ability to set a default quota set.

### CODE REVIEW

File: p9admin/cli/project.py [hunk 1]
```diff
@@ -66,42 +67,33 @@ def apply_quota_all(quota_name, quota_value, force=False):
...
 
     projects = client.projects()
 
-    for project in projects:
-
-        quota = p9admin.project.get_quota(client, project.id)
+    if defaults:
+        for project in projects:
+            p9admin.project.verified_apply_quota_defaults(client, project)
+            sleep(5)
```
Comment (danielparks) [inline p9admin/cli/project.py]: I think this should probably not be here.

File: p9admin/cli/project.py [hunk 2]
```diff
@@ -66,42 +67,33 @@ def apply_quota_all(quota_name, quota_value, force=False):
...
     client = p9admin.OpenStackClient()
 
-    if "NOVA_URL" not in os.environ:
-        client.logger.critical("NOVA_URL environment variable must be set.  Check README.rst")
+    client.logger.info("Starting application of quotas to all projects")
 
-    validators.quota_name(quota_name)
-    validators.quota_value(quota_name, quota_value)
+    if "OS_NOVA_URL" not in os.environ:
+        client.logger.critical("OS_NOVA_URL environment variable must be set.  Check README.rst")
```
Comment (danielparks) [inline p9admin/cli/project.py]: `sys.exit` here would be nicer (does `client.logger.critical` exit?)

File: p9admin/project.py [hunk 3]
```diff
@@ -8,8 +9,6 @@
 import requests
 import sys
 
-logger = logging.getLogger(__name__)
```
Comment (danielparks) [inline p9admin/project.py]: Why get rid of this?

File: p9admin/project.py [hunk 4]
```diff
@@ -113,51 +112,66 @@ def apply_quota(client, project_id, quota_name, quota_value):
...
 
+def apply_quota_defaults(client, project_id):
+    """
+    Apply a quota to an existing project
+    """
+    config = configparser.ConfigParser()
+    config.read('conf/defaults.ini')
+
+    for key in config["DEFAULT"]:
+        print("Applying key {} with value {}".format(key, config["DEFAULT"][key]))
```
Comment (danielparks) [inline p9admin/project.py]: Could you make this use the `logger`? Maybe with debug? This is going to generate a lot of output for applying to all projects.

File: p9admin/project.py (hunk 3 above)
Comment (danielparks) [reply p9admin/project.py]: Not asking you to change it back, just curious.

Comment (mikkergimenez) [reply p9admin/project.py]: Huh not sure, I wasn't sure what the pattern was, whether or not we were using client.logger or assigning a new logger in each class.  I noticed in some cases it was using the client.logger so I was trying to make it consistent.

File: p9admin/cli/project.py (hunk 2 above)
Comment (mikkergimenez) [reply p9admin/cli/project.py]: No, not sure where that came from, I thought I was copying a pattern I saw.  Will change

File: p9admin/project.py (hunk 3 above)
Comment (danielparks) [reply p9admin/project.py]: I was intending to create a logger for each module, but as long as it's consistent then +1

**Comment by danielparks** [comment]
I reopened this as https://github.com/puppetlabs/p9-admin/pull/11 to fix the merge conflicts. Sorry, meant to merge this but I forgot about it.

**Pull request closed without merging**